test/gui/shared/steps/steps.py

```diff
@@ -542,169 +444,62 @@ def step(context, receiver, resource):
 
 @Then('the error text "|any|" should be displayed in the sharing dialog')
 def step(context, fileShareContext):
+    shareItem = SharingDialog()
+    errorText = shareItem.getErrorText(shareItem.ERROR_SHOWN_ON_SHARING_DIALOG)
```

Comment:
you don't have to pass the arg here since this variable is available in the class itself

Reply:
Yes you are right. I did this so that I could you it for some other steps as well that didn't work. Will change it now.